### PR TITLE
Updates to the signing and packaging projects

### PR DESCRIPTION
**Pull request by jgoshi**

### CODE REVIEW

File: VSInsertion/Signing/SignFiles.csproj [hunk 1]
```diff
@@ -46,7 +46,6 @@
     <ItemGroup>
       <FilesToSign Include="@(FilesToCopy->'$(OutDir)%(Filename)%(Extension)')">
         <Authenticode>Microsoft</Authenticode>
```
Comment (rchiodo) [inline VSInsertion/Signing/SignFiles.csproj:48]: Why did you remove this?

Comment (jgoshi) [reply VSInsertion/Signing/SignFiles.csproj:48]: Because I'm experimenting. Looking at the logs of the build definition I'm writing and the build definition of the existing VCLS signing that we manually trigger, they both say strong name authentication cannot be done (there's a warning) but then the exes get submitted to some server for signing. I'm trying to remove things we don't need to reduce confusion in the future and want to see if this removes that warning.

**Review by rchiodo — APPROVED** [review]
